bashtop

```diff
@@ -132,6 +137,13 @@ menu_quit_selected=(
 "║═╬╗║ ║ ║ ║  "
 "╚═╝╚╚═╝ ╩ ╩  ")
 
+menu_update_selected=(
+"*┬ ┬┌─┐┌┬┐┌─┐┌┬┐┌─┐"
+" │ │├─┘ ││├─┤ │ ├┤ "
+" └─┘┴  ─┴┘┴ ┴ ┴ └─┘"
+)
+
+declare -r  remote_src="https://raw.githubusercontent.com/aristocratos/bashtop/master/$(basename $0)" 
```

Comment:
This will fail to get remote if local file isn't named "bashtop"

Reply:
You're absolutely right. I forgot that detail.